STEP_Project/popup.js

```diff
@@ -0,0 +1,120 @@
+var pageType = undefined;
+
+/** Enum for page types. */
+const PAGE_TYPES = {
+  ORG_LIST_PAGE: 0,
+  SETTINGS_PAGE: 1, // includes onoff page
+  OTHER: 2,
+};
+
+(function()
+{
+  checkTabUrl();
+  addRemoveButton = document.getElementById("addRemove");
+  injectContent(addRemoveButton);
+}()
+);
+
+/**
+ * Checks url of the current tab to detect settings or OU list page
+ * @param {string} givenurl - url of current page
+ */
+function findPagecase(givenurl)
```

Comment:
nit: should we rename this to findPageType or getPageType ? If yes, rename at other places too.

Reply:
Change was made in the next PR